homeassistant/components/yale_smart_alarm/__init__.py

```diff
@@ -1 +1,57 @@
 """The yale_smart_alarm component."""
+from __future__ import annotations
+
+import asyncio
+
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN, LOGGER
+from .coordinator import YaleDataUpdateCoordinator
+
+PLATFORMS = ["alarm_control_panel"]
+
+
+async def async_setup_entry(hass, entry):
+    """Set up Yale from a config entry."""
+    hass.data.setdefault(DOMAIN, {})
+    title = entry.title
+
+    coordinator = YaleDataUpdateCoordinator(hass, entry=entry)
+
+    await coordinator.async_refresh()
+    if not coordinator.last_update_success:
+        raise ConfigEntryNotReady
+
+    hass.data[DOMAIN][entry.entry_id] = {
```

Comment:
Are you expecting to add more data at `hass.data[DOMAIN][entry.entry_id]`? If not, you can set the coordinator directly on it instead of creating a new dictionary.

Reply:
It's intended to contain more data as more features are coming in upcoming PRs